homework_27/test_data/variables.py

```diff
@@ -0,0 +1,14 @@
+"""VARIABLES FOR API TESTS"""
+
+url = "https://alexqa.netlify.app/.netlify"
+token = ("eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9."
+         "eyJ1c2VySWQiOiIxMTY4NzI0NzY5MjY4MDM1N"
+         "jM4MDIiLCJpYXQiOjE3MjM4ODE0MzcsImV4cC"
+         "I6MTcyMzg4NTAzN30.KadleyNmrDry_1bq8aV"
+         "jmyRlca66j-0XaDCO3p89_HY")
```

Comment:
Not really save to store secret information in the repository

Reply:
Fixed. Now token stored in config.json file and this file is added to .gitignore.
Also added get_token function that retrieves token from that file.
File structure:
{
    "api_token": " ... "
}